File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/workbook/NamesCountWorkbook.java
package com.aspose.cells.cloud.examples.workbook;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class NamesCountWorkbook {

    public static void main(String... args) throws IOException {
        // ExStart: get-names-count-workbook
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);
            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "Sample1.xlsx";
            Path inputFile = Utils.getPath(NamesCountWorkbook.class, input);

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

            com.aspose.cells.model.NamesResponse np = cellsApi.GetWorkBookNames(input, null, null);

            System.out.println(np.getNames().getCount());
        } catch (Exception e) {
            e.printStackTrace();
        }

        // ExEnd: get-names-count-workbook

    }
}

File: Examples/Node.js/SDK/workbook/NamesCountWorkbook.js
var fs = require('fs'),
    assert = require('assert'),
    utils = require('../utils.js'),
    cells = utils.cellsApi,
    storage = utils.storageApi;


var inputFile = "sample1.xlsx";
var inputPath = utils.getPath(__filename, inputFile);


storage.PutCreate(inputFile, null, null, inputPath, function (responseMessage) {
    cells.GetWorkBookNames(inputFile, null, null, function(responseMessage) {
            assert.equal(responseMessage.status, 'OK');
        console.log("Total Workbook Count : " + responseMessage.body.Names.Count);
        });
    });
